controllers/storageclaim_controller.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/red-hat-storage/ocs-client-operator/pkg/utils"
 
 	"github.com/go-logr/logr"
+	groupsnapapi "github.com/kubernetes-csi/external-snapshotter/client/v6/apis/volumegroupsnapshot/v1alpha1"
```

Comment:
use `v8` client

Reply:
It requires go  >= 1.22.0. Should I go ahead with the upgrade?